frontend/packages/citizen-frontend/src/decisions/ApplicationLink.tsx

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import InlineButton from '@evaka/lib-components/src/atoms/buttons/InlineButton'
+import { faFileAlt } from '@evaka/lib-icons'
+import { noop } from 'lodash'
+import React from 'react'
+import { useTranslation } from '~localization'
+
+export const ApplicationLink = React.memo(function ApplicationLink({
+  applicationId
+}: {
+  applicationId: string
+}) {
+  const t = useTranslation()
+
+  return (
+    <a
```

Comment:
Again, use just `Link` instead of `<a>` since this is a link to another route in the application.